common/consts/consts.go

```diff
@@ -25,7 +25,12 @@ const (
 	CurrentNamespaceEnvVar                         = "CURRENT_NS"
 	DefaultNamespace                               = "monitoring"
 	OTLPPort                                       = 4317
+	OTLPPortStr                                    = "4317"
```

Comment:
Maintainers will have to remember to maintain both `OTLPPort` & `OTLPPortStr`.
Why not using `strconv.Itoa(OTLPPort)` where needed?